fixed issue in sample manifest

ChristopherDosin opened the pull request:
fixed error during the first app installation and app activation  
```
Message: [ERROR 1871] Element 'license': This element is not expected. Expected is ( version ).
```
and 
```
ERROR 1871] Element 'permissions': Missing child element(s). Expected is one of ( read, create, update, delete ).
```

klarstil commented:
Hey @jkrzefski thanks for the contribution. I reviewed your changes and merged your PR. The changes will be reflected on GitHub in a few after the bot syncs repository with Github.

Best regards

mitelg commented:
merged with https://github.com/shopware/shopware/commit/44552d7521a7a8b855e0fcb29300bca7bf77076b

ChristopherDosin committed:
fixed issue in sample manifest

fixed `Message: [ERROR 1871] Element 'license': This element is not expected. Expected is ( version ).` and `ERROR 1871] Element 'permissions': Missing child element(s). Expected is one of ( read, create, update, delete ).`

bneumann97 commented:
Hey @ChristopherDosin, thanks for your contribution 💙
